docs/DeveloperGuide.md

```diff
@@ -239,6 +239,68 @@ _{more aspects and alternatives to be added}_
 _{Explain here how the data archiving feature will be implemented}_
 
 
+### Edit Feature
+
+#### What is this feature about? 
+The `edit` feature allows the user to change the attributes of the Candidate in the system though the CLI.
+This feature has been enhanced based on the initial implementation of the `edit` command in `AddressBook3`.
+
+#### How is this feature implemented?
+
+The proposed edit mechanism is facilitated by `EditCommand`. It is supported by `EditCommandParser` where it will retrive the attributes that can be edited. 
+
+The user is able to edit key attributes of the Candidate, such as 
+* Phone Number
+* ApplicationStatus 
+* InterviewStatus
+
+Based on the fields that are edited by the user, the EditCommandParser will retreive the information, and update the attributes accordingly.
+There are also some attributes that are dependent on another attribute, and we will introduce `Triggers` to update the attributes automatically.
```

Comment:
A veryvery minor (optional) note, just in case the `Triggers` will be implemented by v1.3b, may also indicate it is 'proposed' or a 'wip' :)

Reply:
Ah okok !! Actually the trigger is already implemented and working for 1.3 already ! But definitely would still need to refine, so I'll just put 'wip' first!